docs/kyma-environment-broker/04-01-cls-architecture.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Cloud Logging Service Architecture
+type: Architecture
+---
+
+Cloud Logging Service (CLS) is a managed logging service for shipping logs from Runtimes. It uses the Elastic Stack (open source logging platform Elasticsearch, Logstash, Kibana) to store, parse, and visualize the application log data coming from Kubernetes applications.
+
+ Kyma environment broker provisions a CLS instance for a global account, creates a binding and passes the CLS credentials to the provisioner as SKR overrides so that fluent bit in the SKR can be properly configured to push the logs to the CLS instance. The architecture diagram below explains the complete process
```

Comment:
[Our docu](https://kyma-project.io/docs/control-plane/kyma-environment-broker/#documentation-content) capitalize this. 
Not sure about the "can be" - is this optional? If not, I'd use "is configured. Or are further user actions needed? Then I suggest "so that you can configure..."
```suggestion
 Kyma Environment Broker provisions a CLS instance for a global account and creates a binding. It passes the CLS credentials to the provisioner as SKR overrides, so that fluent bit in the SKR is properly configured to push the logs to the CLS instance. The following architecture diagram explains the complete process:
```